README.md

```diff
@@ -3,6 +3,19 @@ Contains configuration and sources to build node.js
 
 The main usecase right now is building `node.js@18+` with `glibc@2.17`, which is required for some older platforms. (More context: https://github.com/nodejs/unofficial-builds/pull/69)
 
+## How to generate a new build? 
+We have a kibana-flavored buildkite job that does most of the work for building these distributables, by running the scripts in this repo. You can find the job here: https://buildkite.com/elastic/kibana-custom-node-dot-js-builds - [configured here](https://github.com/elastic/kibana-buildkite/blob/main/pipelines/kibana-custom-node-build.tf).
+
+### Build on a branch
+You can either create a new branch, where you set the new default values for the Node & re2 versions, and run a new build on that branch (or get the PR merged, and run a build on `main`). See [this PR](https://github.com/elastic/kibana-custom-nodejs-builds/pull/8) for reference.
+
+### Build with override parameters
```

Comment:
Can we add an extra note about what is needed to be done regards RE2 bumps? 

https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/patch_native_modules_task.ts#L43

https://github.com/elastic/kibana/blob/main/src/dev/build/tasks/patch_native_modules_task.ts#L53

Reply:
okay!